MobileBuy/buy/src/main/java/com/shopify/buy/model/ProductTag.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *   The MIT License (MIT)
+ *
+ *   Copyright (c) 2015 Shopify Inc.
+ *
+ *   Permission is hereby granted, free of charge, to any person obtaining a copy
+ *   of this software and associated documentation files (the "Software"), to deal
+ *   in the Software without restriction, including without limitation the rights
+ *   to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *   copies of the Software, and to permit persons to whom the Software is
+ *   furnished to do so, subject to the following conditions:
+ *
+ *   The above copyright notice and this permission notice shall be included in
+ *   all copies or substantial portions of the Software.
+ *
+ *   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *   THE SOFTWARE.
+ */
+package com.shopify.buy.model;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Represents tag information attached to the product
+ */
+public class ProductTag extends ShopifyObject {
```

Comment:
This class not only wraps String it extends the Shopify object that has id field. So we have id, handle. And this class is used only internally, API returns List of Strings not List of ProductTags. Take a look at the response from server API

`
{
"tags": [3]
0:  {
"id": 12562217347
"title": "Vintage"
}-
1:  {
"id": 12562217283
"title": "summer"
}-
2:  {
"id": 12562217219
"title": "cotton"
}-
-
}
`